docs/docs/reference/project-files/connectors.md

```diff
@@ -28,7 +28,7 @@ Select + Add -> Data -> Connect an OLAP engine
 
 :::tip A note on OLAP engines
 
-You can have multiple OLAP engines in a single project and have each dashboard run on a different engine using the olap_connector parameter directly on a dashboard.
+You can have multiple OLAP engines in a single project and have each dashboard run on a different engine using the `connector` parameter directly on a dashboard.
```

Comment:
Rereading this again, I'm thinking we should also say "using the connector parameter directly within the dashboard YAML" with a link to the dashboard YAML reference page? WDYT?

Reply:
I agree, i should've checked these before merging, will make the changes tomorrow